### PR TITLE
Minor rewording of safe-ffi-wrapper exercise.

### PR DESCRIPTION
**Pull request by njr0**
For readability.

### CODE REVIEW

**Review by mgeisler — APPROVED** [review]
Sounds good, thanks!

File: src/exercises/day-3/safe-ffi-wrapper.rs [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 // ANCHOR: solution
 // ANCHOR: ffi
+
+use tempfile::tempfile;  // requires tempfile = "3" in Cargo.toml dependencies
```
Comment (mgeisler) [inline src/exercises/day-3/safe-ffi-wrapper.rs:18]: We test the solution using the [`Cargo.toml` in `src/exercises`](https://github.com/google/comprehensive-rust/blob/main/src/exercises/Cargo.toml) and this line is not actually needed — the test code below uses the `tempfile` crate directly.

If you would want to import `tempfile::TempDir` for the tests, then you should add the `use tempfile::TempDir` line _inside_ `mod tests`. This way it becomes possible to distinguish between regular dependencies and dev-dependencies. The latter are only used by `cargo test` and won't burden other users of the code.

T

File: src/exercises/day-3/safe-ffi-wrapper.rs [hunk 2]
```diff
@@ -14,6 +14,9 @@
...
+
```
Comment (mgeisler) [inline src/exercises/day-3/safe-ffi-wrapper.rs:19]: Let me take it out so we can merge the PR directly:

```suggestion
```

Comment (mgeisler) [inline src/exercises/day-3/safe-ffi-wrapper.rs:19]: Let me take it out so we can merge the PR directly:

```suggestion
```

**Comment by njr0** [comment]
Sorry, I see what you mean now. I added the `use` first, and that didn't fix it, so then I updated the Cargo.toml. But I didn't notice the warning:
```sh
warning: unused import: `tempfile::tempfile`
 --> src/main.rs:1:5
  |
1 | use tempfile::tempfile;
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

With you now!

**Comment by mgeisler** [comment]
The only missing piece here is the Google CLA — we're not allowed to import code without a signed CLA.

**Pull request merged**